pipelines/pipelineMaxEnt.json

```diff
@@ -0,0 +1,991 @@
+{
```

Comment:
I suggest to remove some parameters that could be put as input to the pipeline. For example, the species.